techniques/system/server-roles/1.0/component-check.st

```diff
@@ -36,22 +36,18 @@ bundle agent root_component_check
       "service[rudder_slapd][name]"                    string => "slapd";
       "service[rudder_slapd][binary]"                  string => "/opt/rudder/libexec/slapd";
       "service[rudder_slapd][initscript]"              string => "rudder-slapd";
-      "service[rudder_slapd][check_on_relay_server]"   string => "false";
 
       "service[rudder_jetty][name]"                    string => "jetty";
       "service[rudder_jetty][binary]"                  string => ".*java.*/opt/rudder/jetty7/start.jar";
       "service[rudder_jetty][initscript]"              string => "rudder-jetty";
-      "service[rudder_jetty][check_on_relay_server]"   string => "false";
 
       "service[rudder_reports][name]"                  string => "postgresql";
       "service[rudder_reports][binary]"                string => "postgres: writer process";
       "service[rudder_reports][initscript]"            string => "postgresql";
-      "service[rudder_reports][check_on_relay_server]" string => "false";
 
       "service[apache][name]"                          string => "apache";
       "service[apache][binary]"                        string => "${apache_dist_specific_name}";
       "service[apache][initscript]"                    string => "${apache_dist_specific_name}";
-      "service[apache][check_on_relay_server]"         string => "true";
```

Comment:
These parameters were used to tell the ```generic_process_check_process``` method whether or not to check this process on a relay server, back in the past. Now, we manage this condition directly in this file, as you can see below.